### PR TITLE
update index

### PR DESCRIPTION
**Pull request by jdorando**

### CODE REVIEW

File: solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java [hunk 1]
```diff
@@ -133,17 +133,17 @@ public String getName() {
    * @return the id
    */
   public int getId() {
```
Comment (mnilsson23) [inline solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java]: should rename get/sets too

File: solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java [hunk 2]
```diff
@@ -204,7 +204,7 @@ public String getName() {
     }
 
     public int getId() {
-      return Feature.this.id;
+      return Feature.this.index;
```
Comment (mnilsson23) [inline solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java]: getIndex?

**Comment by jdorando** [comment]
code review update

File: solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java [hunk 3]
```diff
@@ -200,11 +200,11 @@ public FeatureWeight(Query q, IndexSearcher searcher,
     }
 
     public String getName() {
-      return Feature.this.name;
+      return Feature.this.getName();
     }
 
     public int getId() {
-      return Feature.this.id;
+      return Feature.this.getIndex();
```
Comment (mnilsson23) [inline solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/Feature.java]: I meant the function to change from getId to getIndex like you did above

**Comment by jdorando** [comment]
code review update

**Pull request merged**